test/forms/inputs/date-input.test.js

```diff
@@ -9,6 +9,8 @@ const noop = () => {}
 const input = { name, value }
 const error = 'input error'
 
+const sleep = () => new Promise((r) => setTimeout(() => r(), 5000))
```

Comment:
👀

Reply:
Oh noooo, how'd this get left in there 😭